tests/dummy/app/routes/application.js

```diff
@@ -3,13 +3,13 @@ import Route from '@ember/routing/route';
 import config from '../config/environment';
 import { inject as service } from '@ember/service';
 import { scheduleOnce } from '@ember/runloop';
-import steps from '../data';
+import { steps as defaultSteps } from '../data';
 
 export default Route.extend({
   tour: service(),
   disableScroll: true,
   defaults: {
-    classes: 'shepherd-element shepherd-theme-arrows',
+    classes: 'shepherd-element shepherd-theme-arrows custom-default-class',
```

Comment:
`shepherd-element` is applied by Shepherd itself, so we don't need that class anywhere.

Reply:
✂️